doc/Caliptra.md

```diff
@@ -233,116 +233,116 @@ In accordance with OCP Attestation specification [Reference 8](#ref-8), devices
+|Side channel - timing|Attacker observes the elapsed time of different sensitive operations.|Includes attacks where the attacker actively stimulates the sensitive operations while timing.|
+|Cryptographic analysis|Attacker observes plaintext, ciphertext, related data, or immediate values in cryptography to overcome cryptographic controls.|<p>Includes all practical cryptanalysis attacks.</p><p>Assumes NIST-unapproved algorithms provide no security (for example, SHA-1, Single DES, ChaCha20).</p><p>Assumes any cryptographic algorithm that provides less than 128 bits of security (as determined by NIST SP 800-57) provides no security.</p><p>Excludes quantum computer attacks. This exclusion will be removed soon.</p>|
 
-### Trust Boundaries
+### Trust boundaries
 
-Following diagram establishes trust boundaries for the discussion of threat modeling. Caliptra based SoCs are expected to have Caliptra as silicon RoT, platform or SoC security engine to orchestrate SoC security needs and rest of the SoC.
+The following figure shows trust boundaries for the discussion of threat modeling. SOCs based on Caliptra are expected to have Caliptra as silicon RoT, and are expected to have a platform or SoC security engine to orchestrate SoC security needs and the rest of the SoC.
 
 Trust levels of Caliptra and the SoC security engine are not hierarchical. These two entities are responsible for different security aspects of the chip.
 
-*Figure 1: Caliptra Trust Boundaries*
+*Figure 1: Caliptra trust boundaries*
 
 ![Caliptra_trust_boundary](./images/Caliptra_trust_boundary.png)
 
-### Caliptra Interactions
+### Caliptra interactions
 
-The “Caliptra Core” blocks consume the Tcc and Tcw Trust level components.  This boundary includes crypto accelerators, hardware key sequencer, key vault, Caliptra microcontroller, ROM and subsystem interconnects.  The Caliptra Core provides deterministic Caliptra behavior.  The Caliptra Subsystem includes the Tc, Tcw, Tse and Trs, this subsystem extends Trust level into the SoC subsystem and allows for customization of the SoC initialization.  For information on Caliptra Core and Caliptra Subsystem, see section: [Caliptra Profiles](#caliptra-profiles)
+The Caliptra Core blocks consume the Tcc and Tcw trust level components.  This boundary includes crypto accelerators, hardware key sequencer, key vault, Caliptra microcontroller, ROM, and subsystem interconnects.  The Caliptra Core provides deterministic Caliptra behavior.  The Caliptra subsystem includes the Tc, Tcw, Tse, and Trs. The Caliptra subsystem extends the trust level into the SoC subsystem and allows for customization of the SoC initialization.  For information on Caliptra Core and the Caliptra subsystem, see [Caliptra Profiles](#caliptra-profiles).
```

Comment:
Something to fix when we update the diagram, but I don't see "Tcc" in [Caliptra_trust_boundary.png](https://github.com/chipsalliance/Caliptra/raw/main/doc/images/Caliptra_trust_boundary.png). Should this paragraph just replace "Tcc" with "Tc"? Or vice versa? @vsonims

Reply:
out of scope for this PR. created issue